### PR TITLE
ncm-network: always enable the network service

### PR DESCRIPTION
**Pull request by stdweird**
based on #1372 
Relying on only ncm-chkconfig can give issues in case of failures with ncm-chkconfig (and network service is not even present by default on EL8)

### CODE REVIEW

File: ncm-network/src/main/perl/network.pm [hunk 1]
```diff
@@ -1983,6 +1983,14 @@ sub Configure
             # check if this was due to addition of resolv_mods / peerdns
             my $no_resolv = $self->make_ifcfg($ifacename, $iface, $ipv6, resolv_mods => 0, peerdns => 0);
             $self->legacy_keeps_state($file_name, $no_resolv, $exifiles);
+        };
+
+        if ($exifiles->{$file_name} == $UPDATED) {
+            # This is a UGent issue: a version with resolv_mods only was once build and deployed.
+            # Adding only the peerdns causes a restart
+            # TODO: get rid of this after a weeks
```
Comment (ned21) [inline ncm-network/src/main/perl/network.pm]: Should this merged as-is if it's a UGent only temp fix?

Comment (stdweird) [reply ncm-network/src/main/perl/network.pm]: @ned21 no, we can get rid of this next week i think. (but we need this in a PR for our internal build tools)

**Comment by jrha** [comment]
I have removed the temporary commit and rebased on master.

**Review by jrha — DISMISSED** [review]
LGTM AFAICT

**Comment by jrha** [comment]
rebased

**Review by jrha — APPROVED** [review]
LGTM

**Review by ned21 — APPROVED** [review]

**Pull request merged**